packages/fx-core/resource/package.nls.json

```diff
@@ -380,6 +380,16 @@
   "core.common.OutlookDesktopClientName": "Outlook desktop client id",
   "core.common.OutlookWebClientName1": "Outlook web access client id 1",
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
+  "core.common.CancelledMessage": "Operation is cancelled.",
+  "core.common.SpecVersionNotSupported": "OpenAPI version lower than 3.0.0 is not supported.",
+  "core.common.NoServerInformation":  "No server information is found in the OpenAPI specification file.",
+  "core.common.MultipleServerInformation": "Multiple server information is found in the OpenAPI specification file.",
+  "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
+  "core.common.MissingOperationId": "Missing operationIds: %s.",
+  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI specification file.",
+  "core.common.AdditionalPropertiesNotSupported": "'additionalProperties' is not supported, and will be ignored.",
+  "core.common.SchemaNotSupported": "'oneOf', 'anyOf', and 'not' schema are not supported: %s.",
+  "core.common.UnknownSchema": "Unknown schema: %s.",
   "core.importAddin.label": "Import an Existing Outlook Add-in",
   "core.importAddin.detail": "Upgrade and add-in project to the latest app manifest and project structure",
```

Comment:
Upgrade and -> Upgrade an?

Alo please check the official doc, how the Outlook Add-In is capitalized. whether it is Outlook Add-In or Add-in or add-in, then update this line and the previous line for it.

Reply:
Thanks, from [doc](https://support.microsoft.com/en-us/office/get-an-office-add-in-for-outlook-1ee261f9-49bf-4ba6-b3e2-2ba7bcab64c8) it is: Add-ins